x-pack/plugins/canvas/canvas_plugin_src/uis/tags/chart.ts

```diff
@@ -4,10 +4,6 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export function Tag(config) {
-  // The name of the tag
-  this.name = config.name;
+import { TagFactory } from '../../../public/lib/tag';
 
-  // color of the tag to display in a list
-  this.color = config.color;
-}
```

Comment:
This diff is just wrong... I converted `public/lib/tag.js` to TS and it should be compared against `public/lib/tag.ts`